_caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md

```diff
@@ -0,0 +1,139 @@
+
+[Uganda Rural Development Training (URDT)  Girls’ School](https://urdt.net/urdt-girls-school-urdtgs/), is an all girl’s school that was established in 2000 in Kagadi district in Western Uganda. The school was instituted to “fill the need for education that is linked to development and women empowerment”. Their pedagogical approach is to employ the 2-generation approach, where parents learn along with their children to “develop a shared vision for their home, analyze their current situation, apply systems thinking, team learning, plan together and learn new skills”. 
+
+The students are viewed as change agents in their home and in the community, as they are encouraged to study while also being equipped through “training to generate sustainable income, health, family cooperation, and peace at home”. This education is transferred back at home through various projects to uplift the family economically, and it transferred through educative theater, workshops and radio programmes - all which results in students as young as seventeen years old being [able to attest](https://www.youtube.com/watch?v=K2opGQ3jQJo&t=153s) that they have managed to build their family a house from revenue they are getting from family farming projects, for example. In contrast to other schools, URDT Girls’ School aims to produce “functionally literate, principled, entrepreneurial and responsible citizens”.
+
+It was probable to the author that cultural differences between how URDT teaching staff and the visiting lecturers engaged, taught, learned, communicated, and prioritized activities - made a typical data collection process for the needs inadequate. Often, assessment data collection tools consist of online or paper surveys and interviews that are designed in light of the existing available data. Obtaining available data was a substantial challenge in an agriculture-focused institution where teaching, assessment and communication methods did not prioritize written accounts or data storage in ways common to the visiting lecturers. The teaching staff’s experience with tech tools was non-uniform and sparse. Their teaching [methodology](https://urdt.net/our-methodology/) was one that prioritized group discussions and practical hands-on work to enable swift rural transformation. As such, it was decided to run focus group interviews in order to get as much information from the variation of responses given, and account for the gaps in hearing and understanding that had been identified earlier in the interactions of  the volunteer lecturers and URDT staff. 
```

Comment:
@mutsacc 
Please check this link https://urdt.net/our-methodology/.  Error: 'This site can’t be reached`